modules/dataframe/src/properties/filterlistproperty.cpp

```diff
@@ -84,7 +94,7 @@ FilterListProperty::FilterListProperty(std::string_view identifier, std::string_
             InvalidationLevel::InvalidOutput, PropertySemantics::Text);
     };
 
-    {
+    if (supportedFilters & FilterType::StringItem) {
```

Comment:
you don't have to change all of these, but I think 
`supportedFilters.contains(FilterType::StringItem)`
is much more readable.